### PR TITLE
[CEC] Added advances settings for action when TV goes standby

### PR DESCRIPTION
**Pull request by filimonic**
Added advances settings for action when TV goes standby : Pause
playback, Stop playback, Exit Kodi

### CODE REVIEW

**Comment by Jalle19** [comment]
@opdenkamp

**Comment by milokmet** [comment]
+1 this will be a very useful feature for all kodi users! My kids sometimes didn't stop the playback before turning off the TV.

**Comment by Jalle19** [comment]
Would need a rebase before it can be considered again.

File: system/peripherals.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <setting key="send_inactive_source" type="bool" value="1" label="36025" order="5" />
     <setting key="cec_standby_screensaver" type="bool" value="0" label="36009" order="6" />
     <setting key="cec_wake_screensaver" type="bool" value="1" label="36010" order="7" />
-    <setting key="standby_pc_on_tv_standby" type="enum" value="13011" label="36029" order="8" lvalues="36028|13005|13011" />
+    <setting key="standby_pc_on_tv_standby" type="enum" value="13011" label="36029" order="8" lvalues="36028|13005|13011|13009|36043|36045" />
```
Comment (bkuhls) [inline system/peripherals.xml]: Afaics 36043 should be 36044 (LOCALISED_ID_STOP)

36043 means something else: https://github.com/xbmc/xbmc/blob/master/addons/resource.language.en_gb/resources/strings.po#L16296

**Comment by opdenkamp** [comment]
Rebased, cleaned up and included in #10775: 41621bcc99be98f0b2e9e4845564894a16348ba3

**Pull request closed without merging**